openstf/feature_engineering/weather_features.py

```diff
@@ -0,0 +1,332 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+""" This module contains all wheather related functions used for feature engineering.
+
+"""
+import numpy as np
+import pandas as pd
+
+# Set some (nameless) constants for the Antoine equation:
+A: float = 6.116
+M: float = 7.6
+TN: float = 240.7
+# Set some constants
+TORR: float = 133.322368  # 1 torr = 133 Pa
+# 1.168 is the mass of 1 m^3 of air on sea level with standard pressure.
+D: float = 1.168
+
+
+def calc_saturation_pressure(temperature: float or np.ndarray) -> float or np.ndarray:
+    """Function that calculates the water vapour pressure from the temperature
```

Comment:
@JanMaartenvanDoorn -> Calculate the water saturation pressure. Or something similar